cmd/podman/system/events.go

```diff
@@ -99,25 +99,28 @@ func eventsCmd(cmd *cobra.Command, _ []string) error {
 		errChannel <- err
 	}()
 
-	for event := range eventChannel {
-		switch {
-		case event == nil:
-			// no-op
-		case doJSON:
-			jsonStr, err := event.ToJSONString()
-			if err != nil {
-				return err
+	for {
```

Comment:
The changes here look unrelated to the commit.

Reply:
They are not, for loop or event channel is blocking. Therefore we can never read the error channel.